app/services/replication_transfer_updater.rb

```diff
@@ -83,7 +83,7 @@ def request_storage
       if params[:fixity_value] == correct_fixity
         record.update(store_requested: true)
       else
-        record.cancel!("fixity_reject")
+        record.cancel!("fixity_reject", params[:cancel_reason_detail])
       end
```

Comment:
In slack chat we seemed to arrive at logging some details and adding a detail that simply refers to the logs (maybe add a date or other information about accessing the log).